### PR TITLE
Move cherry picking functionality over to Prow

### PR DESCRIPTION
**Pull request by jennybuckley**
#7332

Moves experiment/cherrypicker under prow/external-plugins as suggested in https://github.com/kubernetes/test-infra/issues/7332#issuecomment-374402511

Is this really enough to get it running in k/k though?
I think it also needs a deployment and service, etc, like external-plugins/needs-rebase has.

### CODE REVIEW

**Comment by BenTheElder** [comment]
> I think it also needs a deployment and service, etc, like external-plugins/needs-rebase has.

It'll need those, we'll want to fix up the build to look like the needs-rebase and add it to the deployment bundle. I think those can be other PR(s) though.

/cc @kargakis

**Comment by BenTheElder** [comment]
needs a `hack/update-bazel.sh` to update the build.

**Comment by kargakis** [comment]
Please, leave a breadcrumb in experimenta;/cherrypicker that points to the new location.

File: experiment/cherrypicker/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This project has moved to [https://github.com/kubernetes/test-infra/tree/master/prow/external-plugins/cherrypicker](https://github.com/kubernetes/test-infra/tree/master/prow/external-plugins/cherrypicker)
```
Comment (jennybuckley) [inline experiment/cherrypicker/README.md]: @kargakis Is this okay for the breadcrumb?

Comment (kargakis) [reply experiment/cherrypicker/README.md]: yeah, thanks

**Pull request merged**